tasks/perun_openldap_config.yml

```diff
@@ -132,51 +132,95 @@
   when: ansible_facts.services['slapd'] is not defined
   service_facts:
 
-- name: "Add limits to /etc/security/limits.conf"
-  lineinfile:
-        path: /etc/security/limits.conf
-        state: present
-        line: "{{ item }}"
-  loop:
-    - 'openldap soft nofile 20480'
-    - 'openldap hard nofile 40960'
-  notify: Reload limits
-
-- name: "Add system limit to /etc/sysctl.conf"
-  lineinfile:
-        path: /etc/sysctl.conf
-        state: present
-        line: 'fs.file-max = 500000'
-  notify: Reload sysctl
-
-- name: "Create systemd override config directory if it does not exist"
-  file:
-        path: /etc/systemd/system/slapd.service.d
-        state: directory
-        mode: '0755'
-
-- name: "Add LimitNOFILE to /etc/systemd/system/slapd.service.d/override.conf"
-  blockinfile:
-        path: /etc/systemd/system/slapd.service.d/override.conf
-        create: yes
-        block: |
-          [Service]
-          LimitNOFILE=40960
-  notify: Reload systemd
-
-- name: "Check open file limit using ulimit"
-  command: bash -c 'ulimit -n'
-  register: ulimit_result
-  changed_when: false
-
-- name: "Check system open file limit using sysctl"
-  command: sysctl fs.file-max
-  register: sysctl_result
-  changed_when: false
-
-- name: "Display results of limit checks"
-  debug:
-        msg:
-          - "Current open file limit (ulimit -n): {{ ulimit_result.stdout }}"
-          - "System max open files limit (fs.file-max): {{ sysctl_result.stdout }}"
 
+- name: "set up limits for max open files for LS instance and display it"
```

Comment:
Do not reference LS instance in a comment/name.

Following tasks will work only when we want to set specific limits. What if we want to restore limits to their system default (removing them from instance vars). You probably need similar logic which will run tasks when `perun_ldap_max_open_files_limits` is false and perform opposite actions (setting default value or removing override file).